.github/workflows/publish-jfrog.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish package to jfrog
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+      # Setup .npmrc file to publish to GitHub Packages
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: 'npm'
+          registry-url: 'https://egym.jfrog.io/egym/api/npm/npm-local/'
+          # Defaults to the user or organization that owns the workflow file
+          scope: '@egym'
+      - run: npm install
+      - run: npm run lint
+      - run: npm run build
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.EGYM_NPM_TOKEN }}
```

Comment:
Will it work using the `secrets.EGYM_NPM_TOKEN` for authenticating against Artifactory?

I believe this contains the token to publish to npmjs, not artifactory.
In github actions there are the following env vars for artifactory:
```
ARTIFACTORY_USERNAME
ARTIFACTORY_PASSWORD
```

That is why we normally add the curl command to get .npmrc file form artifactory